src/main.rs

```diff
@@ -23,7 +23,8 @@ mod xwrap;
 use crate::xwrap::Display;
 
 fn usage(progname: &str, opts: getopts::Options) {
-    let brief = format!("Usage: {progname} [options] [file]");
+    let brief = format!("Usage: {progname} [options] [file]
+                        \n    file: use - as [file] for stdout");
     let usage = opts.usage(&brief);
```

Comment:
Does `opts` not have a way to document free-standing options? Abusing the `brief` for that is weird as `getopts` likely won't be able to do any layout on it.